src/pages/AddNotes.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import "../css/note.css";
+
+const AddNotes = (props) => {
+  const [expand, setExpand] = useState(false);
+  const [note, setNote] = useState({
+    title: "",
+    content: "",
+  });
+
+  const InputEvent = (event) => {
+    const { value, name } = event.target;
+
+    setNote((prevData) => {
+      return {
+        ...prevData,
+        [name]: value,
+      };
+    });
+
+    console.log(note);
+  };
+
+  const addNoteHandler = () => {
+    props.passNote(note);
+    setNote({
+      title: "",
+      content: "",
+    });
+  };
+
+  const expandNoteBox = () => {
+    setExpand(true);
+  };
+
+  const backToShrink = () => {
+    setExpand(false);
+  };
```

Comment:
the naming convention of the functions can be better like, ```showNoteModal, hideNoteModal``` or you can combine it into one function as ``` toggleNoteModal (prev=>!prev)```

Reply:
done!